### PR TITLE
Don't reference MTP in UWP

### PR DESCRIPTION
**Pull request by Evangelink**
Contributes to #4371

Thanks to the report, I have realized we were about to ship a version of MSTest broken for UWP. We are sadly having issues with having acceptance test for UWP because of arcade infra.

### CODE REVIEW

File: src/Adapter/MSTest.TestAdapter/RunConfigurationSettings.cs [hunk 1]
```diff
@@ -2,10 +2,14 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.Diagnostics.CodeAnalysis;
-using System.Globalization;
 using System.Xml;
 
+#if !WINDOWS_UWP
+using System.Globalization;
+
```
Comment (fhnaseer) [inline src/Adapter/MSTest.TestAdapter/RunConfigurationSettings.cs:9]: nit: empty line,

**Review by fhnaseer — DISMISSED** [review]

File: src/Adapter/MSTest.TestAdapter/RunConfigurationSettings.cs (hunk 1 above)
Comment (Evangelink) [reply src/Adapter/MSTest.TestAdapter/RunConfigurationSettings.cs:9]: We have policy to have empty line between groups of usings hence the space. Thanks @fhnaseer

File: src/Adapter/MSTestAdapter.PlatformServices/Interfaces/IConfiguration.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if WINDOWS_UWP
+namespace Microsoft.Testing.Platform.Configurations;
+
+public interface IConfiguration
```
Comment (Youssef1313) [inline src/Adapter/MSTestAdapter.PlatformServices/Interfaces/IConfiguration.cs]: Does this have to be public?

Comment (Evangelink) [reply src/Adapter/MSTestAdapter.PlatformServices/Interfaces/IConfiguration.cs]: I thought it was on some public path but looks like not, thanks for pointing it out. Fixing that.

**Pull request merged**